### ISSUE TITLE
Help Text not showing in Internet Explorer 11

### ISSUE THREAD

**Issue by Curtisbrownlee** [wpninjas/ninja-forms#3389]
- **What actions are you taking?**
  - What had happened was... 
User added "Help Text" onto their fields
- **What are you expecting to occur?**
  - What I was expecting to happen was...
Expected the "Help Text" to display in all browsers
- **What is occurring instead?**
  - What *actually* happened was...
"Help Text" displayed in Chrome, Firefox, and Edge, but not in Internet Explorer 11.  There are no errors in the console.  I did notice that in Internet Explorer, the span tag within the label was missing class attributes of "fa-info-circle nf-help".

**Comment by gocsp** [comment]
The Problem is here: 

[`fieldItem.js`](https://github.com/wpninjas/ninja-forms/blob/master/assets/js/front-end/views/fieldItem.js#L233)
`icon.classList.add( 'fa', 'fa-info-circle', 'nf-help' );`

IE 10 & IE 11 doesn't support multiple arguments.

**Comment by kjohnson** [comment]
PING @wpninjas/developers ^

**Comment by gocsp** [comment]
would you please fix this issue.. it should take 5min.

**Comment by slackday** [comment]
Ran into this problem as well. Temporarily fixed it by including `babel-polyfill` and `classlist-polyfill`.

+1 for fixing this in source

**Comment by gocsp** [comment]
i really don't thought about the polyfill solution, thx! but anyways it will be great if someone take 5min and fix this issue.

**Comment by slackday** [comment]
Oh yeah I should have explained that 😄 

Don't know how exactly you use babel in your project. I use [Sage starter theme](https://github.com/roots/sage) and have a config file for webpack which contained

```
  "entry": {
    "main": [
      "./scripts/main.js",
      "./styles/main.scss"
    ],
    "customizer": [
      "./scripts/customizer.js"
    ]
  },
  ...
```

I included the polyfills after installing with `yarn add babel-polyfill classlist-polyfill` as following

```
  "entry": {
    "main": [
      "babel-polyfill",
      "classlist-polyfill",
      "./scripts/main.js",
      "./styles/main.scss"
    ],
    "customizer": [
      "./scripts/customizer.js"
    ]
  },
  ...
```

**Comment by pankajnarola1995** [comment]
> Oh yeah I should have explained that 
> 
> Don't know how exactly you use babel in your project. I use [Sage starter theme](https://github.com/roots/sage) and have a config file for webpack which contained
> 
> ```
>   "entry": {
>     "main": [
>       "./scripts/main.js",
>       "./styles/main.scss"
>     ],
>     "customizer": [
>       "./scripts/customizer.js"
>     ]
>   },
>   ...
> ```
> I included the polyfills after installing with `yarn add babel-polyfill classlist-polyfill` as following
> 
> ```
>   "entry": {
>     "main": [
>       "babel-polyfill",
>       "classlist-polyfill",
>       "./scripts/main.js",
>       "./styles/main.scss"
>     ],
>     "customizer": [
>       "./scripts/customizer.js"
>     ]
>   },
>   ...
> ```
> Another way to try it can be to just do
> 
> ```
> @import 'babel-polyfill';
> @import 'classlist-polyfill';
> ```
> at your js entry-point

Ohh I see, But the problem is I don't use babel,

I use Wordpress with Grunt

so in my package.json 

```{
  "name": "hiscox",
  "devDependencies": {
    "grunt": "~1.0.0",
    "grunt-contrib-watch": "~1.0.0",
    "grunt-postcss": "^0.9.0",
    "grunt-run": "~0.6.0",
    "grunt-sass": "~1.0.0"
  },
  "dependencies": {
    "autoprefixer": "^8.3.0"
  }```

So here I should add something or ?

**Comment by slackday** [comment]
Yes in your package.json you should add `babel-polyfill` and `classlist-polyfill`. 

Sorry long time since I last used Grunt so I can't give you a working example. But you should probably also add [grunt-babel](https://github.com/babel/grunt-babel) and by looking at their examples you adjust it to your project.

**Comment by pankajnarola1995** [comment]
sorry for disturbing you but I have added the dependencies 

```
{
  "name": "hiscox",
  "devDependencies": {
    "grunt": "~1.0.0",
    "grunt-contrib-watch": "~1.0.0",
    "grunt-postcss": "^0.9.0",
    "grunt-run": "~0.6.0",
    "grunt-sass": "~1.0.0"
  },
  "dependencies": {
    "autoprefixer": "^8.3.0",
    "babel-polyfill":"^6.26.0",
    "classlist-polyfill":"^1.2.0"
  }
}
```

but still I can't see the info icon on IE11

**Comment by gocsp** [comment]
You have have [enqueue the script into wordpress](https://developer.wordpress.org/reference/functions/wp_enqueue_script/), there are so many different possible ways - it is not possible so answer them all here.
